source/ui/BioXAS/BioXASZebraLogicBlockInputInvertedView.cpp

```diff
@@ -0,0 +1,77 @@
+#include "BioXASZebraLogicBlockInputInvertedView.h"
+
+BioXASZebraLogicBlockInputInvertedView::BioXASZebraLogicBlockInputInvertedView(BioXASZebraLogicBlockInput *control, QWidget *parent) :
```

Comment:
The two classes are very similar! But I'm not sure there is an easy way to generalize them. I think the alternative might be to set up mappings and button groups in BioXASZebraLogicBlockView. I thought this way might be clearer, though it's at the expense of some code duplication. @dretrex, what do you suggest?